base/docs/Docs.jl

```diff
@@ -676,18 +676,19 @@ end
 
 
 """
-    undocumented_names(mod::Module; all=false)
+    undocumented_names(mod::Module; private=false)
 
-Return an array of undocumented symbols in `module` (that is, lacking docstrings).
-`all=false` returns only exported symbols; whereas `all=true` also includes
-non-exported symbols, following the behavior of [`names`](@ref). Only valid identifiers
-are included. Names are returned in sorted order.
+Return a sorted array of undocumented symbols in `module` (that is, lacking docstrings).
```

Comment:
```suggestion
Return a sorted vector of undocumented symbols in `module` (that is, lacking docstrings).
```
Array isn't wrong, but vector is more precise.